integration_test/index_test.go

```diff
@@ -45,6 +46,25 @@ func TestKrewIndexAdd(t *testing.T) {
 	}
 }
 
+func TestKrewIndexAddUnsafe(t *testing.T) {
+	skipShort(t)
+	test, cleanup := NewTest(t)
+	defer cleanup()
+	test = test.WithEnv(constants.EnableMultiIndexSwitch, 1).WithIndex()
+
+	cases := []string{"a/b", `a\b`, "../a", `..\a`}
+	expected := "invalid index name"
+
+	for _, c := range cases {
+		b, err := test.Krew("index", "add", c, constants.IndexURI).Run()
+		if err == nil {
+			t.Fatalf("%q: expected error", c)
+		} else if !strings.Contains(string(b), expected) {
+			t.Fatalf("%q: output doesn't contain %q: %q", c, expected, string(b))
```

Comment:
Small nit: `"%q: output doesn't contain %q: %q"` -> `"%q: output doesn't contain '%q': %q"`. This way it doesn't read as `a/b: output doesn't contain invalid index name...` but as `a/b: output doesn't contain 'invalid index name'...`

Reply:
%q always adds `"` around the string, so this is not an issue.

Reply:
TIL :smile: